src/Datadog.Trace/SpanTypes.cs

```diff
@@ -37,6 +37,21 @@ public static class SpanTypes
         /// </summary>
         public const string GraphQL = "graphql";
 
+        /// <summary>
+        /// The span type for a message client integration.
+        /// </summary>
+        public const string MessageClient = "queue";
+
+        /// <summary>
+        /// The span type for a message consumer integration.
+        /// </summary>
+        public const string MessageConsumer = "queue";
+
+        /// <summary>
+        /// The span type for a message producer integration.
+        /// </summary>
+        public const string MessageProducer = "queue";
```

Comment:
Is there an RFC about these types? It's weird to me that we aren't differentiating between clients/consumers/producers

Reply:
This distinction is made in the `span.kind` tag. Valid values are `client`, `server`, `consumer`, and `producer`.

Reply:
I was following Java's example in terms of constant names, but I like Lucas's idea to just collapse them into one constant

Reply:
Collapsed to one constant in cec0449fe55dc1a6234f351f8ef6d6a542186d9e . And then I followed it up with another commit to fix a build break because I forgot to change the one usage of it 😅